samples/excel/16-custom-functions/web-call-function.yaml

```diff
@@ -0,0 +1,38 @@
+order: 4
+id: excel-custom-functions-web-call
+name: Web request to GitHub API
+description: Calls the GitHub API to get the star count for an org/user and repository.
+host: EXCEL
+api_set:
+    CustomFunctionsRuntime: 1.1
+script:
+    content: |
+        /**
+         * Gets the star count for a given org/user and repo. Try =GETSTARCOUNT("officedev","office-js")
+         * @customfunction
+         * @param userName string name of org or user
+         * @param repoName string name of the repo
+         * @return number of stars.
+         */
+        async function getStarCount(userName: string, repoName: string) {
+          try {
+            //You can change this URL to any web request you want to work with.
+            const url = "https://api.github.com/repos/" + userName + "/" + repoName;
```

Comment:
Since it is TypeScript you can also use a template:   const url = `https://api.github.com/repos/${userName}/${repoName}`;
It is a little shorter and less " and + to keep track of.

Reply:
@jakobpn excellent suggestion! Thank you!